LethalAPI.Core/Loader/PluginLoader.cs

```diff
@@ -122,12 +111,12 @@ public PluginLoader(LoadMethod loadType)
     /// <summary>
     /// Gets a value indicating whether or not BepInEx is found.
     /// </summary>
-    public static bool BepInExFound { get; private set; }
+    public static bool BepInExFound => LoaderType == LoadMethod.MelonLoader;
```

Comment:
*Should* this be comparing with MelonLoader?

Reply:
Good catch!